common/src/main/java/io/pravega/common/auth/JKSHelper.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.common.auth;
+
+import com.google.common.base.Charsets;
+import java.io.File;
+import java.io.IOException;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FileUtils;
+
+/**
+ * Helper methods for handling JKS and JKS password files.
```

Comment:
Given that this is a public class, can we expand this javadoc to include some reference to JKS?

Reply:
Pointed to JKS documentation for reference.